### ISSUE TITLE
Redirection Forbidden http -> https

### ISSUE THREAD

**Issue by kevinjbayer** [kenpratt/wikipedia-client#28]
This was working great up until this morning. Suddenly, I'm getting a RuntimeError of redirection forbidden.

```
RuntimeError: redirection forbidden:
http://en.wikipedia.org/w/api.php?action=query&format=json&prop=revisions%7Clinks%7Cextlinks%7Cimages%7Ccategories%7Ccoordinates%7Ctemplates&rvprop=content&inprop=url&titles=Getting%20Things%20Done -> 
https://en.wikipedia.org/w/api.php?action=query&format=json&prop=revisions%7Clinks%7Cextlinks%7Cimages%7Ccategories%7Ccoordinates%7Ctemplates&rvprop=content&inprop=url&titles=Getting%20Things%20Done
```

Tried open-uri-redirections (https://github.com/open-uri-redirections/open_uri_redirections) but I couldn't get it working. Anyone know why this suddenly appeared and what a quick solution could be?

**Comment by manuisfunny** [comment]
I got the same error.  I was able to get it working by editing the gem source file client.rb to pass in the open_uri_redirections patch as an option:

client.rb line 47:
```
    def request( options )
      require 'open-uri'
      require 'open_uri_redirections'
      URI.parse( url_for( options ) ).read( "User-Agent" => "Ruby/#{RUBY_VERSION}",  :allow_redirections => :safe)
    end
```

I can check in the change to my fork, then send a pull request later, it you'd like.  I've never done this before, though.

**Comment by baygross** [comment]
Awesome thanks manu.  

I fixed with the following temporary patch in my Gemfile:
gem 'wikipedia-client',:github => 'manuisfunny/wikipedia-client', :branch => 'open_uri_redirection_patch_topic'

And updating my local SSL certificate as described here:
https://gist.github.com/mislav/5026283

**Comment by matthin** [comment]
It's probably best just to force https like so:

```ruby
Wikipedia.Configure {
  protocol "https"
}
```

**Comment by manuisfunny** [comment]
Cool.  Sorry, I'm new to contributing.  Is there anything I should fix on my branch?  Should I just keep it until my pull request is accepted by @kenpratt?  Thanks.

**Comment by matthin** [comment]
I'm not not really a pro at contributing either, but I wouldn't add .idea to .gitignore, or bump the version in a single commit alongside everything else. It's all up to @kenpratt though, he might not even like that I cleared trailing whitespace in my commit.

Also my pull request doesn't actually reference this issue, I just accidentally typed 28 instead of 26.

**Comment by kenpratt** [comment]
Hey, sorry I haven't been responding. I'll make time to merge these PRs and push a new release in the near future.

Short answers: https by defaults sounds good, and I agree with @matthin about pull request structure, if you could amend #29 to not bump the version, and if possible, not make so many changes to the gem versions, that would be great. (And @matthin, +1 to removing trailing whitespace).

**Comment by manuisfunny** [comment]
I made the changes to my pull request for this issue.  Thanks.

**Comment by kenpratt** [comment]
Merged, and pushed a new gem release, 1.4.0.
